lib/rubocop/cop/open_project/no_sleep_in_feature_specs.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module OpenProject
+      # Warn about usage of `sleep N`, where N > 1.
+      class NoSleepInFeatureSpecs < Base
+        MSG = "Avoid using `sleep` greater than 1 second in feature specs. It will reduce overall performance of the test suite. " \
+          "Consider using Capybara's `have_*` matchers or `wait_for` methods for positive matching instead."
+
+        def_node_matcher :sleep_call?, '(send nil? :sleep $...)'
+
+        def on_send(node)
+          return unless feature_spec?(processed_source)
+
+          sleep_call?(node) do |args|
+            if args[0].value > 1
+              add_offense(node, message: MSG)
+            end
+          end
+        end
```

Comment:
In other cops I see in projects like rubocop-rspec, the pattern is more writing `node_predicate?` when using the node matcher in a `if`/`unless`, and `on_node_predicate` when using it with the block notation.

So I think `on_sleep` or `on_sleep_call` would be a better name than `sleep_call?`.